### ISSUE TITLE
Handle dart command lines

### ISSUE THREAD

**Issue by walles** [walles/px#126]
Examples:

```
/usr/local/Caskroom/flutter/3.0.1/flutter/bin/cache/dart-sdk/bin/dart
  --verbosity=error
  --disable-dart-dev
  --snapshot=/usr/local/Caskroom/flutter/3.0.1/flutter/bin/cache/flutter_tools.snapshot
  --snapshot-kind=app-jit
  --packages=/usr/local/Caskroom/flutter/3.0.1/flutter/packages/flutter_tools/.dart_tool/package_config.json
  --no-enable-mirrors
  /usr/local/Caskroom/flutter/3.0.1/flutter/packages/flutter_tools/bin/flutter_tools.dart
```

```
  /usr/local/Caskroom/flutter/3.0.1/flutter/bin/cache/dart-sdk/bin/dart
  devtools
  --machine
  --allow-embedding
```

```
  /usr/local/Caskroom/flutter/3.0.1/flutter/bin/cache/dart-sdk/bin/dart
  --disable-dart-dev
  --packages=/usr/local/Caskroom/flutter/3.0.1/flutter/packages/flutter_tools/.dart_tool/package_config.json
  /usr/local/Caskroom/flutter/3.0.1/flutter/bin/cache/flutter_tools.snapshot
  upgrade
```

`dart --help` says: `Usage: dart <command|dart-file> [arguments]`

So we should show either just the file name or something like `dart upgrade`.

**Comment by walles** [comment]
Also, `=` flags only work with `=` so you can't do `--verbosity error` for example.

We should just take the first word not prefixed with `--`. If it ends with `.dart` or contains a `/`, pretend it's a file name, otherwise a command.

**Comment by walles** [comment]
Fixed, or at least improved by f8ce44ae4d5d63e349f7de1cec8000777ee6ce16.
